test/extended/cli/mustgather.go

```diff
@@ -47,6 +47,10 @@ var _ = g.Describe("[cli] oc adm must-gather", func() {
 			{tempDir, "cluster-scoped-resources", "config.openshift.io", "schedulers.yaml"},
 			{tempDir, "namespaces", "openshift-kube-apiserver", "core", "configmaps.yaml"},
 			{tempDir, "namespaces", "openshift-kube-apiserver", "core", "secrets.yaml"},
+			{tempDir, "audit_logs", "kube-apiserver_audit.log"},
+			{tempDir, "audit_logs", "openshift-apiserver_audit.log"},
```

Comment:
I think you want to look for https://github.com/openshift/must-gather/pull/84/files#diff-c8c5bc1e1afd14dc347fa9d9fe1522f3R28 not the files your currently looking for (for the audit logs).

Reply:
openshift-apiserver.audit_logs_listing
kube-apiserver.audit_logs_listing

Should be what we are looking for.